pkg/scripts/issues/README.md

```diff
@@ -41,7 +41,7 @@
 ```shell
   cd file && go run .
 ```
-4. Next you have to analyze generated CSV and assign categories in the `Category` column and resource / data source in the `Object` column (the `GitHub issues buckets` Excel should be used here named as `GitHubIssuesBucket.csv`). Then, you'll be able to use this csv (put it next to the `main.go`) to assign labels to the correct issues.
+4. Next you have to analyze generated CSV and assign categories in the `Category` column and resource / data source in the `Object` column (the `GitHub issues buckets` Excel should be used here named as `GitHubIssuesBucket.csv`). The csv document be of a certain format with the following columns (with headers): "A" column with issue ID (in the format of "#<issue_id>"), "P" column with the category that should be assigned to the issue (should be one of the supported categories: "OTHER", "RESOURCE", "DATA_SOURCE", "IMPORT", "SDK", "IDENTIFIERS", "PROVIDER_CONFIG", "GRANTS", and "DOCUMENTATION"), and the "Q" column with the object type (should be in the format of the terraform resource, e.g. "snowflake_database"). Then, you'll be able to use this csv (put it next to the `main.go`) to assign labels to the correct issues.
```

Comment:
let's add this file (it's easier to understand and adjust)

Reply:
Hmmm, I hope d you will make this CSV much easier and limit it to only needed columns. I don't see a reason to keep the file with all this columns for this script (it complicates it unnecessarily).